src/types.ts

```diff
@@ -25,9 +25,28 @@ export interface Transaction {
   blocktimeMs: number;
 }
 
+// from liquidjs-lib
+interface TxOutput {
+  asset: Buffer;
+  nonce: Buffer;
+  script: Buffer;
+  value: Buffer;
+  rangeProof?: Buffer;
+  surjectionProof?: Buffer;
+}
+
+// from liquidjs-lib
+interface UnblindOutputResult {
+  asset: Buffer;
+  assetBlindingFactor: Buffer;
```

Comment:
How Buffer gets serialized using the window.postMessage?  I recall we use JSON there, so not sure how to best expose these. Maybe we should return string? ie. hex